client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.controller.js

```diff
@@ -123,6 +123,11 @@ angular.module("managerApp").controller("TelecomTelephonyLinePhoneOrderCtrl", fu
             }).$promise.then(function (result) {
                 _.assign(self.line, { getPublicOffer: result.getPublicOffer }, { isAttachedToOtherLinesPhone: result.isAttachedToOtherLinesPhone });
             });
+        }).then(function () {
+            return OvhApiMe.Lexi().get().$promise.then(function (user) {
```

Comment:
you can paralelize this call with previous (`OvhApiTelephony.Line().Lexi().get({...})` :-)